sql/2020/22_HTTP_2/number_of_h2_pushed_resources_and_avg_bytes.sql

```diff
@@ -0,0 +1,28 @@
+# standardSQL
+# Number of HTTP/2 Pushed Resources and Average Bytes
+SELECT 
+  client,
+  COUNT(DISTINCT page) AS num_pages,
+  APPROX_QUANTILES(num_requests, 1000)[OFFSET(500)] AS median_pushed_requests,
+  APPROX_QUANTILES(kb_transfered, 1000)[OFFSET(500)] AS median_kb_transfered,
+FROM (
+ SELECT 
+    percentile,
+    client,
+    page,
+    SUM(CAST(JSON_EXTRACT_SCALAR(payload, "$._bytesIn") AS INT64)/1024) AS kb_transfered,
```

Comment:
I wonder could we use `respSize` here? See https://httparchive.slack.com/archives/C016QLMT0BZ/p1600189006109100 though as many be unreliable for HTTP/2?